benchmarks/100.webapps/120.uploader/python/function.py

```diff
@@ -28,12 +46,12 @@ def handler(event):
     process_time = (process_end - process_begin) / datetime.timedelta(microseconds=1)
     upload_time = (upload_end - upload_begin) / datetime.timedelta(microseconds=1)
     return {
-            'result': {
+            'status': 'success',
```

Comment:
We also need to update the results parsing in `sebs/faas/function.py` and `sebs/{platform}/triggers.py` - make sure that we check for `status`, and then retrieve results from `measurement`.